src/org/labkey/test/components/domain/DomainFieldRow.java

```diff
@@ -314,6 +326,45 @@ public boolean getDateShift()
         return elementCache().dateShiftBox.get();
     }
 
+    public DomainFieldRow setFromFolder(String formatString)
```

Comment:
as a matter of naming convention, it would be nice for these methods to align closely to the keys they set on the fields- thus, lookupSchema, lookupQuery, lookupContainer
...so setFromFolder -> setlookupContainer,
setFromSchema -> setLookupSchema,
setFromTargetTable -> setLookupQuery